Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1753,6 +1753,7 @@ extension BrowserViewController: SearchViewControllerDelegate {
     }
 
     func searchViewControllerDidFinishHighlighting(_ searchViewController: SearchViewController) {
+        searchViewController.resignFirstResponder()
     }
```

Comment:
This is called when the user has typed DOWN, then UP — i.e. focus should move back to the location bar.